web/src/components/profile/ProfileNewJerseyAddress.tsx

```diff
@@ -52,6 +55,12 @@ export const ProfileNewJerseyAddress = (): ReactElement => {
     setIsValidZipCode(!doesFieldHaveError("addressZipCode"));
   };
 
+  const areAddressFieldsDisabled = (): boolean => {
+    return business && isStartingBusiness(business) && business.formationData.completedFilingPayment
+      ? true
+      : false;
```

Comment:
[dust] I think this is equivalent to 
`return business && isStartingBusiness(business) && business.formationData.completedFilingPayment`

Reply:
Because business can be of type `Business | undefined` the ternary is necessary otherwise, we encounter a type issue.

Reply:
The ternary doesn't make sense here. You could do this with [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) to achieve the same result and it would make more sense visually.

```
return (business && isStartingBusiness(business) && business.formationData.completedFilingPayment) ?? false;
```

Reply:
Another pattern that we've followed in our codebase is to store the condition to a named variable that better articulates the condition that we're trying to check for

```typescript
const hasCompletedFilingPayment = business && isStartingBusiness(business) && business.formationData.completedFilingPayment
```
and then using that in the return:

```typescript
return hasCompletedFilingPayment ?? false
```

Reply:
Updated